sections/migrations.js

```diff
@@ -230,7 +230,7 @@ export default [
   {
     type: "text",
     content:
-      "Seed files are created in the directory specified in your knexfile.js for the current environment. A sample seed configuration looks like:"
+      "Seed files are created in the directory specified in your knexfile.js for the current environment. Using the `--timestamp-filename-prefix` flag in `seed:make` command, the filename will be prefixed with a timestamp, like the migration file names. A sample seed configuration looks like:"
```

Comment:
Is there a reason why flag is described both here and in the flag list?

Reply:
The CLI flag is only described here. In the "seeds" section, I've described the "timestampFilenamePrefix" config option. Are 2 different ways to set same behavior.